docs/getting-started/github-apps/github-apps.rst

```diff
@@ -131,7 +131,8 @@ primaryDescriptorPath
 testParameterFiles (optional)
     An array of absolute paths to test parameter files in the Git repository.
 authors (optional)
-    An array of authorship information, requiring at least the ``name`` of each author.
+    An array of authorship information, requiring at least the ``name`` or ``orcid`` of each author. ``orcid`` is the author's ORCID iD, a 16-digit identifier of the form ``xxxx-xxxx-xxxx-xxxx``, where each ``x`` is a digit.
```

Comment:
```suggestion
    An array of authorship information, requiring at least the ``name`` or ``orcid`` of each author. ``orcid`` is the author's ORCID ID, a 16-digit identifier of the form ``xxxx-xxxx-xxxx-xxxx``, where each ``x`` is a digit.
```

Reply:
`iD` is on purpose because ORCID refers to it as `ORCID iD`: https://info.orcid.org/brand-guidelines/#the-orcid-name